sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -1,18 +1,13 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { bcs, decodeStr, encodeStr } from '@mysten/bcs';
+import { BCS, decodeStr, encodeStr, getSuiMoveConfig } from '@mysten/bcs';
 import { Buffer } from 'buffer';
 import { SuiObjectRef } from './objects';
 
+const bcs = new BCS(getSuiMoveConfig());
```

Comment:
This probably should be exported so that this configured bcs instance can be easily used.

Reply:
It was and is exported down in the file.

Reply:
Ah sorry I missed that!